appinventor/buildserver/src/com/google/appinventor/buildserver/AabZipper.java

```diff
@@ -0,0 +1,65 @@
+package com.google.appinventor.buildserver;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+public class AabZipper {
+  public static boolean zipBundle(File src, File dest, String root) {
+    try {
```

Comment:
Consider using try-with-resources so that these streams are cleaned up correctly. Right now if an error is thrown the streams are not closed.